scripts/check_ecr_findings

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -u
+
+payload="{\"repository\": \"$1\", \"imageTag\": \"$2\"}"
+aws lambda invoke --function-name ecr-scan-findings --payload "$payload" scan_response.json || exit 1
+totalFindings="$(jq -r < scan_response.json | jq .totalFindings)"
```

Comment:
```suggestion
totalFindings="$(jq -r ".totalFindings" < scan_response.json)"
```

By the way, what is the effect of `-r` here?

Reply:
I'll test that in CI, but I'm not sure if it'll work. When I was testing the script locally, this is what the Lambda response looked like:
```
"{\"totalFindings\":0}"
```
And with all the permutations of trying to parse it with a single `jq` call, I kept getting:
```
jq: error (at <stdin>:0): Cannot index string with string "totalFindings"
```

Reply:
Ah, I get it now. The example helps a lot, thank you.

`fromjson | .totalFindings` can do it: https://jqplay.org/s/gFVmrlh22x

Reply:
Thank you! I added the change.